### ISSUE TITLE
Ctrl + Shift + ] Shortcut not working on Windows 10

### ISSUE THREAD

**Issue by snoozxy** [hundredrabbits/Left#123]
Hey guys, loving the style of this application. 
I seem to have found an issue on the two windows 10 machines I'm using where the Ctrl + Shift + ] shortcut doesn't work like it should, though this wasn't present on a mac I borrowed to test.

I can still use Ctrl + Shift + [ to move back a file, and both functions work from the 'navigation' tab in the toolbar, but the next file shortcut isn't working.

**Comment by neauoire** [comment]
We'll have a look :) Stay tuned

**Comment by idimitrijevic** [comment]
I'm also facing issues with quick inserts shortcuts - none of them work :/
Really love the app

**Comment by neauoire** [comment]
Inserts don't work on Windows? 
When you press `ctrl+i` & `ctrl+d` nothing happens?
Can you open an issue for the specific problem?

**Comment by idimitrijevic** [comment]
Yes, sorry :/
I will open an issue
Thanks for the quick response!

**Comment by elliottcandrew** [comment]
any news on this?

ctrl + shift + ] is also not working for me on win10

**Comment by nbfrk** [comment]
Also having this issue on Linux

**Comment by Lucaffo** [comment]
Also having this issue on Windows Today. 
It would be an awesome feature, still not find how and why. 

I see the code for 10 minutes and find that: 

![image](https://user-images.githubusercontent.com/55745404/123510654-08185300-d67d-11eb-85ab-e879f9e8951a.png)

Next_page (The function that need to be called), it's binded in Events.js only when a file is dropped in. 

There is no binding in the file for ctrl+shift+] / [ 

![image](https://user-images.githubusercontent.com/55745404/123510719-5f1e2800-d67d-11eb-8606-91889e624752.png)

**Comment by Lucaffo** [comment]
Ok i've realized now a fact.

Which keyboard layout did you use? Because most non American Layout have '[' that can be used only if you press ctrl+alt. 

if the shortcut require a ctrl+maiusc(Aka shift)+] it should be ctrl+alt+maiusc+] this is equals to curve brakets { } in most non america keyboards lol

**Comment by sdrwtf** [comment]
@Lucaffo you're right, this behavior is depending on the not US keyboard layout, same problem here with a German layout. This also affects the Toggle Navigation shortcut which is `Ctrl + \` by default, but there is an easy fix (or at least workaround).

You can edit the keyboard shortcuts within the `desktop/sources/index.html` file, I've made the following changes and it works fine on Linux:

```js
/* Navigation with Page-Up and Page-Down keys, ~ line 71 */
left.controller.add('default', 'Navigation', 'Next File', () => { left.navi.next_page() }, 'CmdOrCtrl+Shift+PageDown')
left.controller.add('default', 'Navigation', 'Prev File', () => { left.navi.prev_page() }, 'CmdOrCtrl+Shift+PageUp')
left.controller.add('default', 'Navigation', 'Next Marker', () => { left.navi.next_marker() }, 'CmdOrCtrl+PageDown')
left.controller.add('default', 'Navigation', 'Prev Marker', () => { left.navi.prev_marker() }, 'CmdOrCtrl+PageUp')
```

```js
/* Navigation Toggle with Super (windows) key, ~ line 76 */
left.controller.add('default', 'View', 'Toggle Navigation', () => { left.navi.toggle() }, 'CmdOrCtrl+Super+S')
```
After editing the `index.html` I've rebuild Left with `npm rebuild`, actually I'm not sure if this is necessary but it worked for me.

There is just one small quirk: in the menu the super key is not displayed in the shortcut info, but that's only cosmetic (and maybe only missing in the German translation file):

![image](https://user-images.githubusercontent.com/34947398/125478348-b39a480f-efc5-41d8-8783-a7727788e3c2.png)

@neauoire maybe this should be included in the readme? Could save a few recurring questions...
